w3af/core/controllers/dependency_check/platforms/current_platform.py

```diff
@@ -30,9 +30,10 @@
 from .openbsd import OpenBSD5
 from .suse import SuSE
 from .default import DefaultPlatform
+from .LinuxMint17 import LinuxMint17
```

Comment:
according to python's best practices filenames should be lower-case, this means that this line should look like:

```
from .linuxmint17 import LinuxMint17
```